### DISCUSSION TITLE
Kanboard Version, where can I see it?

### DISCUSSION THREAD

**Discussion by ZencoreCZ** [kanboard/kanboard#5244] [Q&A] [answered]
Old version:
![obrazek](https://user-images.githubusercontent.com/9350942/231560140-f9d94273-fd7f-492e-8a36-a0ed7178bda6.png)

And now in Kanboard 1.2.28
![obrazek](https://user-images.githubusercontent.com/9350942/231560262-92e1d5d2-a598-4182-a4b6-234fe66ed104.png)

**Comment by fguillot** [comment, marked as answer]
You can see it in your own screenshot in the configuration section.

![231560262-92e1d5d2-a598-4182-a4b6-234fe66ed104](https://user-images.githubusercontent.com/323546/231641114-5b6eb423-8db0-4b17-a94c-39466db64f6f.png)


You will see the release version if you download a tagged version from GitHub:

![Screenshot 2023-04-12 20 22 43](https://user-images.githubusercontent.com/323546/231640890-7c730b5a-980a-4531-bf2d-de3ffe812fa6.png)
